src/probeinterface/io.py

```diff
@@ -1263,12 +1264,190 @@ def write_imro(file: str | Path, probe: Probe):
         f.write("".join(ret))
 
 
+def read_spikegadgets(file: str | Path) -> ProbeGroup:
+    """
+    Find active channels of the given Neuropixels probe from a SpikeGadgets .rec file.
+    SpikeGadgets headstages support up to three Neuropixels 1.0 probes (as of March 28, 2024),
+    and information for all probes will be returned in a ProbeGroup object.
+
+    Within the SpikeConfiguration header element (sconf), there is a SpikeNTrode element
+    for each electrophysiology channel that contains information relevant to scaling and
+    otherwise displaying the information from that channel, as well as the id of the electrode
+    from which it is recording ('id').
+
+    Nested within each SpikeNTrode element is a SpikeChannel element with information about
+    the electrode dynamically connected to that channel.  This contains information relevant
+    for spike sorting, i.e., its spatial location along the probe shank and the hardware channel
+    to which it is connected.
+
+    Excerpt of a sample SpikeConfiguration element:
+
+    <SpikeConfiguration chanPerChip="1889715760" device="neuropixels1" categories="">
+        <SpikeNTrode viewLFPBand="0"
+            viewStimBand="0"
+            id="1384"  # @USE: The first digit is the probe number; the last three digits are the electrode number
+            lfpScalingToUv="0.018311105685598315"
+            LFPChan="1"
+            notchFreq="60"
+            rawRefOn="0"
+            refChan="1"
+            viewSpikeBand="1"
+            rawScalingToUv="0.018311105685598315"  # For Neuropixels 1.0, raw and spike scaling are identical
+            spikeScalingToUv="0.018311105685598315"  # Extracted when reading the raw data
+            refNTrodeID="1"
+            notchBW="10"
+            color="#c83200"
+            refGroup="2"
+            filterOn="1"
+            LFPHighFilter="200"
+            moduleDataOn="0"
+            groupRefOn="0"
+            lowFilter="600"
+            refOn="0"
+            notchFilterOn="0"
+            lfpRefOn="0"
+            lfpFilterOn="0"
+            highFilter="6000"
```

Comment:
and until L1324

I would remove this from the docstring. You can add a comment in the code.

Reply:
OK.  Before I had it above the loop over that element.  Should I move it back there?